src/ducks/apps/components/AppInstallation.jsx

```diff
@@ -35,6 +36,11 @@ class AppInstallation extends Component {
       getChannel(prevProps.app.source) === channel
     const succeed = justInstalled || justSwitchedChannel
     if (succeed && typeof onSuccess === 'function') {
```

Comment:
The alert does not depend on the `onSuccess` being a function:
```
if (succeed) {
  if (app.type === APP_TYPE.WEBAPP) {
    Alerter.success(t('app_modal.install.message.install_success'), {
      duration: 3000
    })
 }
 if (typeof onSuccess === 'function') {
   onSuccess
 }
}
```
is better imho.

By the way, `onSuccess` could be validated as function with PropTypes and we could save one test.